src/lib/validation/utils/assignHandlerForTrigger.js

```diff
@@ -45,17 +45,16 @@ export default function assignHandlerForTrigger (triggerButton, form) {
       invalidCustomErrorFields.forEach(field => {
         dplan.notify.notify('error', Translator.trans(field.validationMessage))
       })
-      const nonEmptyFieldNames = [...new Set(
-        validatedForm.invalidFields
-          .map(field => field.getAttribute('data-dp-validate-error-fieldname'))
-          .filter(Boolean)
-      )]
 
       if (invalidCustomErrorFields.length === 0 || invalidCustomErrorFields.length < validatedForm.invalidFields) {
+        const nonEmptyFieldNames = validatedForm.invalidFields
+            .map(field => field.getAttribute('data-dp-validate-error-fieldname'))
+            .filter(Boolean)
+        const nonEmptyUniqueFieldNames = nonEmptyFieldNames.filter((field, idx) => nonEmptyFieldNames.indexOf(field) === idx)
```

Comment:
I think we can extract that to an own small method since the exact same Code is used twice.

Reply:
would it be also ok?  -->  https://github.com/demos-europe/demosplan-ui/pull/634/commits/8847423146fad341075cdadde131bd52b7763110#diff-fdc53ff4f95d479db51dcfdf0f2706e9b4974251ed0b258d11a27da73051073bR53